content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -64,7 +64,20 @@ Some use cases for primitive shapes include meshing, gizmos, bounding volumes, c
 [Primitive RFC]: https://github.com/bevyengine/rfcs/blob/main/rfcs/12-primitive-shapes.md
 [collection of primitives]: https://dev-docs.bevyengine.org/bevy/math/primitives/index.html
 
-### Meshing
+### Rendering
+
+Now we're taking a closer look at how to visualize the primitives both in 2D
+and in 3D spaces. In this section, we delve into the rendering process with
+meshes and gizmos. You can check out at all supported rendering capabilities
+that are available by taking a look at the following linked example.
+
+![On the left: A cuboid rendered with gizmos. It consists of 12 white lines. On the right: A cuboid rendered with meshes. It consists of 6 white faces.](cuboids.png)
+
+![On the left: A torus rendered with gizmos. It consists of many small rings, all connected by 4 big rings. On the right: A torus rendered with meshes. A shape that looks like a donut.](tori.png)
+
+[Link to the primitive showcase example](https://bevyengine.org/examples/Math/Math%20Primitives)
```

Comment:
Imo there's a bit too much metatext here. It could be a bit more direct, and also embed the example link and name directly into the paragraph. The link is also wrong I think (examples on the website use the file name kebab-cased)

Something like this?
```suggestion
Primitive shapes can be rendered using both meshes and gizmos. In this section, we'll take a closer look the new APIs.

Below, you can see a cuboid and a torus rendered using meshes and gizmos. You can check out at all primitives that can be rendered in the new [Rendering Primitives] example.

![On the left: A cuboid rendered with gizmos. It consists of 12 white lines. On the right: A cuboid rendered with meshes. It consists of 6 white faces.](cuboids.png)

![On the left: A torus rendered with gizmos. It consists of many small rings, all connected by 4 big rings. On the right: A torus rendered with meshes. A shape that looks like a donut.](tori.png)

[Rendering Primitives]: https://bevyengine.org/examples/Math/render-primitives
```